tool/lib/iso15765_2.py

```diff
@@ -69,15 +69,20 @@ def clear_filters(self):
         """Remove arbitration ID filters"""
         self._set_filters(None)
 
-    def send_message(self, data, arbitration_id):
+    def send_message(self, data, arbitration_id, force_extended=False):
         """
         Transmits a message using 'arbitration_id' and 'data' on 'self.bus'
 
         :param data: Data to send
         :param arbitration_id: Arbitration ID to use
         :return: None
         """
-        msg = can.Message(arbitration_id=arbitration_id, data=data)
+        if arbitration_id <= ARBITRATION_ID_MAX:
```

Comment:
If I am reading your intent correctly, the multi-line check and assignment could simply be expressed like this: 
`is_extended = force_extended or arbitration_id > ARBITRATION_ID_MAX`

Reply:
I couldn't have put it better myself. Looks good.